Apps/Database/src/Delegates/IUserProfileDelegate.cs

```diff
@@ -67,6 +68,14 @@ public interface IUserProfileDelegate
         /// <returns>A DB result which encapsulates the return object and status.</returns>
         DBResult<List<UserProfile>> GetUserProfiles(IList<string> hdIds);
 
+        /// <summary>
+        /// Fetches Hdid from the database.
```

Comment:
Nit: technically, it returns the whole profile for users that match the query.

Reply:
Addressed.